app/views/layouts/application.html.erb

```diff
@@ -5,14 +5,23 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
-
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
     <%= javascript_include_tag "application", "data-turbo-track": "reload", defer: true %>
```

Comment:
Should the javascript be at the end?

Reply:
not highlighted therefore it is not a change on this line